package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "screwdriver-executor-j5s",
+  "version": "1.0.0",
+  "description": "Jenkins Executor plugin for Screwdriver",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "eslint .",
+    "test": "jenkins-mocha --recursive"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:screwdriver-cd/executor-j5s.git"
+  },
+  "homepage": "https://github.com/screwdriver-cd/executor-j5s",
+  "bugs": "https://github.com/screwdriver-cd/executor-j5s/issues",
+  "keywords": [
+    "screwdriver",
+    "yahoo"
+  ],
+  "license": "BSD-3-Clause",
+  "author": "Min Zhang <minzhang@andrew.cmu.edu>",
+  "contributors": [
+    "Cynthia Li <cynthialix@gmail.com>",
+    "Dao Lam <daolam112@gmail.com>",
+    "Darren Matsumoto <aeneascorrupt@gmail.com>",
+    "Jeremiah Wuenschel <jeremiah.wuenschel@gmail.com>",
+    "Min Zhang <minzhang@andrew.cmu.edu>",
+    "Noah Katzman <nbkatzman@gmail.com>",
+    "Peter Peterson <jedipetey@gmail.com>",
+    "St. John Johnson <st.john.johnson@gmail.com",
+    "Tiffany Kyi <tiffanykyi@gmail.com>"
+  ],
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "eslint": "^2.10.2",
+    "eslint-config-screwdriver": "^1.0.0",
+    "eslint-plugin-import": "^1.8.0",
+    "eslint-plugin-jsx-a11y": "^1.2.2",
+    "eslint-plugin-react": "^5.1.1",
+    "jenkins-mocha": "^2.6.0",
+    "mockery": "^1.7.0",
+    "sinon": "^1.17.4"
+  },
+  "dependencies": {
+    "async": "^2.0.1",
+    "circuit-fuses": "^1.0.1",
+    "jenkins": "^0.18.0",
+    "request": "^2.72.0",
+    "screwdriver-executor-base": "^1.0.0"
```

Comment:
Hmm yeah. Either do it in this PR or do it immediately after this. It should be done before implementing other methods.